_sass/minimal-mistakes/_notices.scss

```diff
@@ -108,31 +111,86 @@
   @include notice($danger-color);
 }
 
-
 .notice-highlight {
   background-color: $background-block;
   padding: 1em;
 
   h2 {
     margin-top: 1em;
-
   }
 }
 
 /* Quotes in notice blocks */
 
 .notice blockquote {
   font-size: 1.5em;
-  margin: 5em 2em;
+  margin: 7em 2em;
   max-width: 100%;
 
   &.highlight-quote {
+    margin: 7em .5em;
+    padding: 0;
+    max-width: $medium;
+
     p {
-      margin-bottom: 3px;
+      margin-bottom: 1rem;
+    }
+    &::after {
+      margin: 2rem;
     }
 
     footer {
       text-align: right;
     }
   }
 }
+
+/* Notice elements with figures */
+.notice {
```

Comment:
there are a lot of styles that should belong in a single scss sheet that are in blanket pyos- scss files (from before we broke out from the theme). we will need to slowly combine and reorganize things to make the code easier to maintain.